src/main/resources /changelog/db.changelog-master.yaml

```
@@ -0,0 +1,38 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: your_name
+      changes:
+        - createTable:
+            tableName: book
```

Have you checked and run you app? 
```suggestion
            tableName: books
```